momento-sdk/src/main/java/momento/sdk/internal/GrpcChannelOptions.java

```diff
@@ -0,0 +1,38 @@
+package momento.sdk.internal;
+
+import io.grpc.netty.shaded.io.grpc.netty.NettyChannelBuilder;
+import java.util.concurrent.TimeUnit;
+import momento.sdk.config.transport.GrpcConfiguration;
+
+public class GrpcChannelOptions {
+  // The default value for max_send_message_length is 4mb.  We need to increase this to 5mb in order
+  // to
+  // support cases where users have requested a limit increase up to our maximum item size of 5mb.
+  public static final int DEFAULT_MAX_MESSAGE_SIZE = 5_243_000; // bytes
+
+  public static final boolean DEFAULT_KEEPALIVE_WITHOUT_STREAM = true;
+  public static final int DEFAULT_KEEPALIVE_TIME_MS = 5000; // milliseconds
+  public static final int DEFAULT_KEEPALIVE_TIMEOUT_MS = 1000; // milliseconds
+
+  public static NettyChannelBuilder GrpcOptionsFromGrpcConfig(
```

Comment:
I'm not actually sure if it's idiomatic/acceptable to pass around builder objects like this, open to revising this in a different way if needed

Reply:
I think it's probably okay, I'd just change this signature a bit. You're not returning any grpc options, so I'd probably just call it something like `applyGrpcConfigurationToChannelBuilder`. (Note that in Java, method names always start with a lower case letter, unlike .NET.)

Also since this function is being called purely for side effects I don't think there's any reason to return the builder object back out at the end.